featuretools/computational_backends/feature_set_calculator.py

```diff
@@ -130,7 +130,8 @@ def progress_callback(*args):
             if missing_ids:
                 default_df = self.generate_default_df(instance_ids=missing_ids,
                                                       extra_columns=df.columns)
-                df = df.append(default_df, sort=True)
+
+                df = default_df.append(df, sort=True)
```

Comment:
So this results in using the default feature values + missing instance IDs to create additional rows.
Would it make more sense to not create this? It seems incorrect to add this data...

Reply:
Since this isn't directly related to the initial but, will create a new issue to track.